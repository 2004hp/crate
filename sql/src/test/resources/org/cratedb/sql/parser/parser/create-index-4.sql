create index if not exists idx1 on table1 using fulltext(c1)