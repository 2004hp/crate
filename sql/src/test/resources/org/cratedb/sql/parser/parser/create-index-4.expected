org.cratedb.sql.parser.parser.CreateIndexNode@68e6ff0d
name: idx1
statementType: CREATE INDEX
unique: false
indexName: idx1
tableName: table1
indexMethod: fulltext
indexColumns: 
indexProperties: null
existenceCheck: IF_NOT_EXISTS
	
	org.cratedb.sql.parser.parser.IndexColumnList@1572e449

	[0]:	
	org.cratedb.sql.parser.parser.IndexColumn@29e97f9f
	columnName: c1
	tableName: null
	ascending
